.github/workflows/deploy_on_release.yml

```diff
@@ -44,11 +44,40 @@ jobs:
       with:
         verbose: true
 
-  publish-versioned-website:
-    name: Publish versioned website
+  check-versions:
     needs: package-deploy-pypi
-    uses: ./.github/workflows/reusable_website.yml
+    name: Check if major or minor version changed
+    runs-on: ubuntu-latest
+    outputs:
+      major_minor_changed: ${{ steps.compare.outputs.major_minor_changed }}
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+        fetch-tags: true
+        ref: ${{ github.sha }}
+    - name: Check if major or minor version changed
```

Comment:
When would that not be the case? This workflow is executed upon creating a GitHub release, so this should involve updating at least the minor version? Or is the idea that there may be some post-release update bugfix or the like?

Reply:
There is also the patch versions. v<major>.<minor>.<patch> The idea is to not release website for the patch versions (primarily for Ax, since we'll follow semver there after 1.0), since these are not intended to include any new features, but only minimal bug / documentation fixes.